### PR TITLE
[examples] Add 5-a-side templefile

### PR DESCRIPTION
**Pull request by willjvsmith**
* Adds a 5-a-side templefile

This does highlight a few shortcomings, namely auth roles

### CODE REVIEW

**Review by Lewiky — APPROVED** [review]
cool

**Review by c-harding — APPROVED** [review]
looks good to me (except for the lack of trailing newline 😉)

File: examples/five-a-side.templefile [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ */
+Booking: service {
+  // implicit 'id' field
+  // implicit 'createdBy' field since service does not have auth
+  player: Player;
+  game: Game;
+  #enumerable;
+  #readable(by: all);
+  // implicit #writable(by: this) since project has auth
+}
```
Comment (jaylees14) [inline examples/five-a-side.templefile]: ↩️

**Review by jaylees14 — APPROVED** [review]

**Comment by willjvsmith** [comment]
damn trailing newlines

**Pull request merged**